test/lib/generate/rule-deprecation-test.ts

```diff
@@ -441,4 +441,139 @@ describe('generate (deprecated rules)', function () {
       expect(readFileSync('docs/rules/no-foo.md', 'utf8')).toMatchSnapshot();
     });
   });
+
+  describe('with the old format', function () {
```

Comment:
I think we can remove this test as `replacedBy` is already tested and we're removing the warning.